### PR TITLE
Geopy 1396

### PR DESCRIPTION
**Pull request by benk-mira**
**GEOPY-1396 - Use BaseData class with multi level nested models **

### CODE REVIEW

File: geoapps_utils/driver/data.py [hunk 1]
```diff
@@ -39,28 +39,28 @@ class BaseData(BaseModel):
     title: str
     workspace_geoh5: Optional[Workspace] = None
 
-    @classmethod
-    def _parse_input(cls, input_data: dict[str, Any]) -> dict[str, Union[dict, Any]]:
+    @staticmethod
+    def model_to_dict(
```
Comment (domfournier) [inline geoapps_utils/driver/data.py:43]: Maybe name it something more descriptive? `collect_input_from_dict`?

**Review by domfournier — DISMISSED** [review]
Simple. See suggestion for naming.

**Comment by domfournier** [comment]
You can abandon 3.9 here as well.

File: geoapps_utils/driver/data.py (hunk 1 above)
Comment (benk-mira) [reply geoapps_utils/driver/data.py]: done

**Review by domfournier — DISMISSED** [review]
Noice

**Comment by domfournier** [comment]
@andrewg-mira can you force this PR through? We will also need another alpha tag after

**Review by domfournier — APPROVED** [review]
Great, forcing through coverage until we rafactor params

**Pull request merged**